### PR TITLE
BD-3295 Add 'Metrics by segments' report docs

### PR DESCRIPTION
**Pull request by rachel-feinberg**
### Summary

My PR...

### Related PRs, issues, or features (optional)

#BD-3295

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+---
+nav_title: Metrics by Segments
+article_title: Metrics by segments
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:3]:
```suggestion
article_title: Metrics by Segments
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+- Unique recipients
+- Unique email clicks
+
+All rate metrics use unique email recipients as the denominator.
+
+#### Definitions
+
+- “Upon receipt” refers to purchase events or revenue that occurred within your specified conversion window, after users received the specified campaigns or Canvases. 
+
+- “Upon click” refers to the purchase events or revenue that occurred after the purchase events, within your specified conversion window, after users clicked the specified campaigns or Canvases.
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:105]:
```suggestion
- “Upon receipt” refers to purchase events or revenue that occurred within your specified conversion window, after users received the specified campaigns or Canvases. 
- “Upon click” refers to the purchase events or revenue that occurred after the purchase events, within your specified conversion window, after users clicked the specified campaigns or Canvases.
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+- Unique non-machine opens
+- Unique clicks
+- Unsubscribes
+- Bounces
+- Soft bounces
+- Complaints
+
+To view this report, you must specify the following variables in the **Variables** tab:
+- **Metrics:** Select one of the metrics by which to rank your results
+- **Number of reports:** Select top or bottom results and the number of results, such as Top 10 or Bottom 15
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:163]:
```suggestion
- **Number of reports:** Select top or bottom results and the number of results, such as top 10 or bottom 15
```
Disregard if it should be "Top 10" and "Bottom 15" 🫡

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+- **Message type:** Specify if your results are campaigns, Canvases, or Canvas steps
+
+#### Results
+
+Your results will show the top (or bottom) campaigns, Canvases, or Canvas steps that you selected. For example, if you selected the top 10 campaigns for click rate, your results will show the top 10 campaigns ordered from highest to lowest click rate. Your columns will display all of the email engagement metrics for each row (campaigns, Canvases, or message steps).
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:168]:
```suggestion
Your results will show the top (or bottom) campaigns, Canvases, or Canvas steps that you selected. For example, if you selected the top 10 campaigns for click rate, your results will show the top 10 campaigns ordered from highest to lowest click rate. Your columns will display all of the email engagement metrics for each row (campaigns, Canvases, or Message steps).
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+- Unique purchases upon receipt
+- Revenue upon receipt
+- Unique purchases upon click
+- Revenue upon click
+- Unique recipients
+- Unique email clicks
+
+To view this report, you must specify the following variables in the **Variables** tab:
+- **Metrics:** Select one of the metrics by which to rank your results
+- **Number of reports:** Select top or bottom results and the number of results, such as Top 10 or Bottom 15
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:192]:
```suggestion
- **Number of reports:** Select top or bottom results and the number of results, such as top 10 or bottom 15
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+- **Message type:** Specify if your results are campaigns, Canvases, or Canvas steps
+- **Conversion window:** The number of days after email receipt or click to which Braze will attribute purchases or revenue 
+
+#### Definitions
+
+- “Upon receipt” refers to purchase events or revenue that occurred within your specified conversion window, after users received the specified campaigns or Canvases. 
+
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:199]:
```suggestion
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+- “Upon click” refers to the purchase events or revenue that occurred after the purchase events, within your specified conversion window, after users clicked the specified campaigns or Canvases.
+
+For example, let's say a segment contains 10 users and five of them made a purchase after receiving your email. If one of those five made a purchase after clicking your email, your "unique purchases upon receipt" rate would be 50% and your "unique purchases upon click" rate would be 10%.
+
+#### Results
+
+Your results will show the top (or bottom) campaigns, Canvases, or Canvas steps that you selected. For example, if you selected top 10 campaigns for "revenue upon click", your results will show the top 10 campaigns ordered from highest to lowest "revenue upon click". Your columns will display all of the purchase metrics for each row (campaigns, Canvases, or message steps).
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:206]:
```suggestion
Your results will show the top (or bottom) campaigns, Canvases, or Canvas steps that you selected. For example, if you selected the top 10 campaigns for "revenue upon click", your results will show the top 10 campaigns ordered from highest to lowest "revenue upon click". Your columns will display all of the purchase metrics for each row (campaigns, Canvases, or Message steps).
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+  - Segments
+  - Reports
+  
+---
+
+# Metrics by segments
+
+> Use [Query Builder]({{site.baseurl}}/user_guide/data_and_analytics/query_builder/) report templates to break down performance metrics for campaigns, Canvas, variants, and steps by segments.
+
+![The Segment breakdown page contains a SQL editor, a side panel with tabs for Variables, Available Data Tables, Query History, and the AI Query Builder, and a results section.]({% image_buster /assets/img_archive/segment_breakdown.png %})
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:17]:
```suggestion
![The Segment breakdown page contains a SQL editor, a side panel with tabs for Variables, Available Data Tables, Query History, and the AI Query Builder, and a results section.]({% image_buster /assets/img_archive/segment_breakdown.png %})
```
Thoughts on moving this image to before the Report templates header? 🤔  Up to you!

**Review by lydia-xie — APPROVED** [review]
looks great! left some feedback 💯

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:17]:
```suggestion
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+To run these reports, do the following:
+1. In **Query Builder**, choose to create a new SQL report via a template. 
+2. Filter templates for those where the metrics include “segment breakdowns”, which are:
+- Email performance metrics by segment
+- Email engagement metrics for variants or steps, by segment
+- Purchases and revenue by segment
+- Purchases and revenue for variants or steps, by segment
+- Push performance by segment
+
+## Report templates
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:30]:
```suggestion

![The Segment breakdown page contains a SQL editor, a side panel with tabs for Variables, Available Data Tables, Query History, and the AI Query Builder, and a results section.]({% image_buster /assets/img_archive/segment_breakdown.png %})

## Report templates
```

File: _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md [hunk 11]
```diff
@@ -0,0 +1,231 @@
...
+  - Segments
+  - Reports
+  
+---
+
+# Metrics by segments
+
+> Use [Query Builder]({{site.baseurl}}/user_guide/data_and_analytics/query_builder/) report templates to break down performance metrics for campaigns, Canvas, variants, and steps by segments.
+
+
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/reporting/data_by_segments.md:17]:
```suggestion
```

**Pull request merged**